desktop/core/src/desktop/settings.py

```diff
@@ -654,23 +654,30 @@ def is_oidc_configured():
 ################################################################
 
 # Insert our custom upload handlers
-file_upload_handlers = [
+if get_chunked_fileuploader():
+  file_upload_handlers = [
+    'hadoop.fs.upload.FineUploaderChunkedUploadHandler',
```

Comment:
Is `FineUploaderChunkedUploadHandler` coming in a different commit?

Reply:
Yes, https://github.com/cloudera/hue/pull/3504